utils.go

```diff
@@ -30,14 +30,11 @@ func getValueByKey(kmsService *cloudkms.Service, storageService *cloudstore.Clie
 	return decryptedValue, nil
 }
 
-// valueOrReadFrom returns the value argument if not empty or the contents of the file argument if empty.
-func valueOrReadFrom(value string, file *os.File) string {
-	if len(value) != 0 {
-		return value
-	}
-	buffer, err := ioutil.ReadAll(file)
+// readFromStdIn tries to read tries to read required input from standard in
+func readFromStdIn() string {
```

Comment:
the old function was a generalization ; why was it replaced? what if we want a future flag "-i" that explictly reads from file?  (ok we can apply generalization later)

Reply:
The generalized function was used in three places with as input os.StdIn. I removed this complexity for now.